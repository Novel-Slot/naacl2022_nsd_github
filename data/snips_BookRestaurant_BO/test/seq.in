i want to bring four people to a place that s close to downtown that serves churrascaria cuisine
can you get me reservations for a highly rated restaurant in seychelles
on june 27  2026 i d like to go to a delaware gastropub
make a reservation at a bakery that has acquacotta in central african republic for five
make me a reservation in south carolina
restaurant in bulgaria this week party for 9 numbers
book a spot for ten at a top-rated  caucasian restaurant not far  from selmer
book a table for 2 at a restaurant in follett
book a brasserie in samoa for four people
find a reservation at a restaurant that serves gougère in laneville with a party of nine
book a bakery for lebanese on january 11th  2032
book a table for 8 at a restaurant that serves far breton
book a table today at a steakhouse for eight that serves sashimi
coon chicken inn restaurant for 1 am for me  clarice and debbie
find a reservation at fish express
book a bar that serves italian-american cuisine neighboring wilson av for one person
book a restaurant for 3 people at eighteen oclock in saint vincent and the grenadines
i would like to eat fast food and have a party of two in kentucky
book a table around london borough of ealing that is highly rated in a gluten free bar
book me a reservation at a bar around juliff for three people that serves bucatini for now
book a highly rated place in in in seven years  at a pub
book a table for ten for breakfast in minnesota
book a restaurant for three on feb  18
please make me reservations somewhere for eight people in foley  nv
i d like to eat at a restaurant around china with a party of 7 anywhere that serves ouzeri
i d like a table for midday at the unseen bean
i want a table for me and my kids in turkey at a neighboring restaurant
i d like to reserve a table at a pub that serves andouillettes within the same area in san marino
book the space aliens grill & bar in hord wy for feb  the twenty-seventh
book a table this evening in saint vincent and the grenadines at a gastropub
food truck in panama for five
book a table for chasity ruiz and mary at the fat duck in puerto rico
i need a table for 1 minute from now at any pub for five around in that also serves fisn n chips
book a spot at the food truck in ma
21 weeks from now elinor  crystal turner and nita want to eat german food at a bar in distant california
book a spot for 7 at an outdoor food court in denmark
i need a reservation for february 27  2020 at a bar that serves paté
i need a reservation for ten at a tavern in west virginia
i want a table for five at a restaurant with latin food in arkansas for 1 hour from now
book a mediterranean restaurant for my sister and i
book me a reservation for eight for the top-rated bakery eleven hours from now in mango
table for 8 at a popular food court
find me a table for 8 people at a nearby al restaurant one minute from now
book me a restaurant please
i would like to book a restaurant for two in 42 weeks from now in wagram
book a restaurant distant from downtown
book a table for nine people in svalbard and jan mayen
book a reservation for clinton street baking company & restaurant distant from downtown
find a restaurant in fm that servec quiche
i want to book a pastelaria cafe in alabama for me and my great grandfather
make a reservation in a popular sicilian bar place nearby  for me only  tomorrow
i need a table for 9
i need a table in ottoville on feb  15th  2029 at gus stevens seafood restaurant & buccaneer lounge
i need a table for five at childs restaurants in brunei
i would like to make a reservation for 2 for brunch
need a table  for party of five for december 26  2040 in the state of mt
book me a restaurant for nine in statham
i d like a table for ten in 2 minutes at french horn  sonning eye
i want to eat close to bowlegs seven years from now
book a popular restaurant of thai cuisine
i d like to eat at the best restaurant
i want to book a restaurant for four around zapata
i want to book a bar in bonaparte palau
can you make reservations at a tea house that serves fettucine
i want to go to 88th st-boyd av or close by and book seats for 10
i want to book a cafe for 3 in fargo
i d like to eat at a popular brasserie in chile with a party of 5
book a table at grecian coffee house for 7 on apr  7th  2024
i m looking for a churrascaria place with wifi that can serve a party of five
book the city tavern in holiday ks
book at table at forest av restaurant close-by for 2 1 second from now
book me a table for 5 at a best rated restaurant in italy
please make reservations in yeager for seven am at a highly rated indian brasserie
book a spot at savoy hotel and grill that is neighboring wisconsin
please book a room in spaghetti warehouse for catalina  delores and brandie mendoza at 12 am
i need to book a top-rated steakhouse this autumn  for 1 around azerbaijan
i would like to book the best food court with persian food within the same area as ok for my ex husband and i
book a restaurant close by my daughters s work location with burrito three years from now
i would like to book a highly rated brasserie with souvlaki neighboring la next week
i need to take three people to eat
book a table in the united states for 10 at the berghoff
i d like to book a brasserie in virginia city  ga
book a spot at a restaurant within walking distance of palau
book a restaurant in donnelly
i d like to eat at an internet restaurant with a party of four
please make reservations for nine at 3 am
book a spot for kelli  jean and i at a pub at elevenses
book a taverna that serves bengali for six at five
table for one somewhere in palco
book me a table for 3 at tkk fried chicken in sri lanka
book a restaurant for one person at 7 am
i d like a reservation at a place in iran for neva  alice and maggie parker
