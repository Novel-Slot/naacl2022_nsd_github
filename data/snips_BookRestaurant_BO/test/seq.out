O O O O B-party_size_number O O O O O O B-spatial_relation O B-poi O O B-restaurant_type O
O O O O O O O B-sort I-sort B-restaurant_type O B-country
O B-timeRange I-timeRange I-timeRange O O O O O O O B-state B-restaurant_type
O O O O O B-restaurant_type O O B-served_dish O B-country I-country I-country O B-party_size_number
O O O O O B-state I-state
B-restaurant_type O B-country B-timeRange I-timeRange O O B-party_size_number O
O O O O B-party_size_number O O B-sort B-cuisine B-restaurant_type B-spatial_relation I-spatial_relation O B-city
O O O O B-party_size_number O O B-restaurant_type O B-city
O O B-restaurant_type O B-country O B-party_size_number O
O O O O O B-restaurant_type O O B-served_dish O B-city O O O O B-party_size_number
O O B-restaurant_type O B-cuisine O B-timeRange I-timeRange I-timeRange
O O O O B-party_size_number O O B-restaurant_type O O B-served_dish I-served_dish
O O O B-timeRange O O B-restaurant_type O B-party_size_number O O B-served_dish
B-restaurant_name I-restaurant_name I-restaurant_name B-restaurant_type O B-timeRange I-timeRange O B-party_size_description I-party_size_description I-party_size_description I-party_size_description
O O O O B-restaurant_name I-restaurant_name
O O B-restaurant_type O O B-served_dish I-served_dish B-spatial_relation B-poi I-poi O B-party_size_number O
O O B-restaurant_type O B-party_size_number O O B-timeRange O O B-country I-country I-country I-country I-country
O O O O O B-restaurant_type I-restaurant_type O O O O O B-party_size_number O B-state
O O O B-spatial_relation B-poi I-poi I-poi I-poi O O B-sort I-sort O O B-cuisine I-cuisine B-restaurant_type
O O O O O O B-restaurant_type B-spatial_relation B-city O B-party_size_number O O O B-served_dish O B-timeRange
O O B-sort I-sort O O B-state B-timeRange I-timeRange I-timeRange O O B-restaurant_type
O O O O B-party_size_number O B-timeRange O B-state
O O B-restaurant_type O B-party_size_number O B-timeRange I-timeRange
O O O O O O B-party_size_number O O B-city B-state
O O O O O O O B-restaurant_type B-spatial_relation B-country O O O O B-party_size_number O O O B-cuisine
O O O O O O B-timeRange O B-restaurant_name I-restaurant_name I-restaurant_name
O O O O O B-party_size_description I-party_size_description I-party_size_description I-party_size_description O B-country O O B-spatial_relation B-restaurant_type
O O O O O O O O O B-restaurant_type O O B-served_dish B-spatial_relation I-spatial_relation I-spatial_relation I-spatial_relation O B-country I-country
O O B-restaurant_name I-restaurant_name I-restaurant_name I-restaurant_name I-restaurant_name O B-city B-state O B-timeRange I-timeRange I-timeRange
O O O O B-timeRange O B-country I-country I-country I-country I-country O O B-restaurant_type
B-restaurant_type I-restaurant_type O B-country O B-party_size_number
O O O O B-party_size_description I-party_size_description I-party_size_description I-party_size_description O B-restaurant_name I-restaurant_name I-restaurant_name O B-country I-country
O O O O O B-timeRange I-timeRange I-timeRange I-timeRange O O B-restaurant_type O B-party_size_number B-spatial_relation B-state O O O B-served_dish I-served_dish I-served_dish
O O O O O B-restaurant_type I-restaurant_type O B-state
B-timeRange I-timeRange I-timeRange I-timeRange B-party_size_description I-party_size_description I-party_size_description I-party_size_description I-party_size_description O O O B-cuisine O O O B-restaurant_type O B-spatial_relation B-state
O O O O B-party_size_number O O B-facility B-restaurant_type I-restaurant_type O B-country
O O O O O B-timeRange I-timeRange I-timeRange O O B-restaurant_type O O B-served_dish
O O O O O B-party_size_number O O B-restaurant_type O B-state I-state
O O O O O B-party_size_number O O B-restaurant_type O B-cuisine O O B-state O B-timeRange I-timeRange I-timeRange I-timeRange
O O B-cuisine B-restaurant_type O B-party_size_description I-party_size_description I-party_size_description I-party_size_description
O O O O O B-party_size_number O O B-sort B-restaurant_type B-timeRange I-timeRange I-timeRange I-timeRange O B-city
O O B-party_size_number O O B-sort B-restaurant_type I-restaurant_type
O O O O O B-party_size_number O O O B-spatial_relation B-state B-restaurant_type B-timeRange I-timeRange I-timeRange I-timeRange
O O O B-restaurant_type O
O O O O O O B-restaurant_type O B-party_size_number O B-timeRange I-timeRange I-timeRange I-timeRange O B-city
O O B-restaurant_type B-spatial_relation O B-poi
O O O O B-party_size_number O O B-country I-country I-country I-country
O O O O B-restaurant_name I-restaurant_name I-restaurant_name I-restaurant_name I-restaurant_name I-restaurant_name B-spatial_relation O B-poi
O O B-restaurant_type O B-state O O B-served_dish
O O O O O B-cuisine B-restaurant_type O B-state O B-party_size_description I-party_size_description I-party_size_description I-party_size_description I-party_size_description
O O O O O B-sort B-cuisine B-restaurant_type O B-spatial_relation O O O B-timeRange
O O O O O B-party_size_number
O O O O O B-city O B-timeRange I-timeRange I-timeRange O B-restaurant_name I-restaurant_name I-restaurant_name I-restaurant_name I-restaurant_name I-restaurant_name I-restaurant_name
O O O O O B-party_size_number O B-restaurant_name I-restaurant_name O B-country
O O O O O O O O B-party_size_number O B-timeRange
O O O O O O B-party_size_number O B-timeRange I-timeRange I-timeRange O O O O B-state
O O O B-restaurant_type O B-party_size_number O B-city
O O O O O O B-party_size_number B-timeRange I-timeRange I-timeRange O B-restaurant_name I-restaurant_name I-restaurant_name I-restaurant_name
O O O O B-spatial_relation O B-city B-timeRange I-timeRange I-timeRange I-timeRange
O O B-sort B-restaurant_type O B-cuisine O
O O O O O O O B-sort B-restaurant_type
O O O O O B-restaurant_type O B-party_size_number B-spatial_relation B-city
O O O O O B-restaurant_type O B-city B-state
O O O O O O B-restaurant_type I-restaurant_type O O B-served_dish
O O O O O B-poi I-poi I-poi O B-spatial_relation O O O O O B-party_size_number
O O O O O B-restaurant_type O B-party_size_number O B-city
O O O O O O O B-sort B-restaurant_type O B-country O O O O B-party_size_number
O O O O B-restaurant_name I-restaurant_name I-restaurant_name O B-party_size_number O B-timeRange I-timeRange I-timeRange
O O O O O B-restaurant_type O O B-facility O O O O O O B-party_size_number
O O B-restaurant_name I-restaurant_name O B-city B-state
O O O O B-poi I-poi B-restaurant_type B-spatial_relation O B-party_size_number B-timeRange I-timeRange I-timeRange I-timeRange
O O O O O B-party_size_number O O B-sort I-sort B-restaurant_type O B-country
O O O O B-city O B-timeRange I-timeRange O O B-sort I-sort B-cuisine B-restaurant_type
O O O O B-restaurant_name I-restaurant_name I-restaurant_name I-restaurant_name O O B-spatial_relation B-state
O O O O O B-restaurant_name I-restaurant_name O B-party_size_description I-party_size_description I-party_size_description I-party_size_description I-party_size_description O B-timeRange I-timeRange
O O O O O B-sort B-restaurant_type B-timeRange I-timeRange O B-party_size_number B-spatial_relation B-country
O O O O O O B-sort B-restaurant_type I-restaurant_type O B-cuisine O B-spatial_relation I-spatial_relation I-spatial_relation I-spatial_relation O B-state O B-party_size_description I-party_size_description I-party_size_description I-party_size_description I-party_size_description
O O B-restaurant_type B-spatial_relation I-spatial_relation B-poi I-poi I-poi I-poi I-poi O B-served_dish B-timeRange I-timeRange I-timeRange I-timeRange
O O O O O O B-sort I-sort B-restaurant_type O B-cuisine B-spatial_relation B-state B-timeRange I-timeRange
O O O O B-party_size_number O O O
O O O O O B-country I-country O B-party_size_number O B-restaurant_name I-restaurant_name
O O O O O O B-restaurant_type O B-city I-city B-state
O O O O O B-restaurant_type B-spatial_relation I-spatial_relation I-spatial_relation O B-state
O O B-restaurant_type O B-city
O O O O O O O B-facility B-restaurant_type O O O O B-party_size_number
O O O O B-party_size_number O B-timeRange I-timeRange
O O O O B-party_size_description I-party_size_description I-party_size_description I-party_size_description O O B-restaurant_type O B-timeRange
O O B-restaurant_type O O B-cuisine O B-party_size_number O B-timeRange
O O B-party_size_number O O B-city
O O O O O B-party_size_number O B-restaurant_name I-restaurant_name I-restaurant_name O B-country I-country
O O B-restaurant_type O B-party_size_number O O B-timeRange I-timeRange
O O O O O O O O O B-country O B-party_size_description I-party_size_description I-party_size_description I-party_size_description I-party_size_description
